includes/wc-admin-update-functions.php

```diff
@@ -76,3 +77,23 @@ function wc_admin_update_0251_remove_unsnooze_action() {
 function wc_admin_update_0251_db_version() {
 	Installer::update_db_version( '0.25.1' );
 }
+
+/**
+ * Convert the "excluded" order status black list into an "included" white list.
+ * See: https://github.com/woocommerce/woocommerce-admin/issues/4050.
+ */
+function wc_admin_update_104_migration_excluded_order_status() {
```

Comment:
Should we come up with some way of auto-triggering the data sync when this occurs or do we think this is a rare enough scenario that users should manually import after updating?

If we do want to trigger the import, we can probably only trigger if there are differences between the two currently synced statuses array.

Reply:
Order status isn't taken into account when syncing - only in report queries.